src/cache/models.rs

```diff
@@ -166,7 +166,7 @@ mod question {
     #[derive(Debug, Default, Serialize, Deserialize)]
     pub struct MetaData {
         pub name: Option<String>,
-        pub params: Option<Param>,
+        pub params: Option<Vec<Param>>,
```

Comment:
This params option was failing in the latest version of the CLI, seems having it as Option<Vec< works fine